insteon_mqtt/device/HiddenDoor.py

```diff
@@ -0,0 +1,652 @@
+        # Bit 3
+        if rawflags & 0b00001000 == 8:
+            LOG.info("\tSend Repeated Closed Commands [Every 5 mins for 50 "
+                     "mins]")
+        else:
+            LOG.info("\tDon't Send Repeated Closed Commands")
+        # Bit 4
+        if rawflags & 0b00010000 == 16:
+            LOG.info("\tLink to FF Group")
+        else:
+            LOG.info("\tDon't link to FF Group")
+        # Bit 5
+        if rawflags & 0b00100000 == 32:
+            LOG.info("\tLED does not blink on transmission")
+        else:
+            LOG.info("\tLED blinks on transmission")
+        # Bit 6
+        if rawflags & 0b01000000 == 64:
+            LOG.info("\tNo Effect")
+        else:
+            LOG.info("\tNo Effect")
+        # Bit 7
+        if rawflags & 0b10000000 == 128:
+            LOG.info("\tProgramming lock on")
+        else:
+            LOG.info("\tProgramming lock off")
+
+        # D6 Heart Beat Interval
+        # 5 minute increments for 1-255.  0 = 24 hours (default)
+        hb_interval = msg.data[5]
+        if hb_interval > 0:
+            hb_interval_minutes = hb_interval * 5
+        else:
+            hb_interval_minutes = 24 * 60 # convert 24 hours to minutes
+        LOG.info("\tHeart beat interval raw level is %s or %s minutes ",
+                 hb_interval, hb_interval_minutes)
+
+        # D7 Low Better Level
+        lb_level = msg.data[6]
+        LOG.info("\tLow battery level is %s", lb_level)
+
+        # D4 Battery Level
+        batt_volt = msg.data[3]
+        LOG.info("Hidden door sensor %s Battery voltage is %s", self.addr,
+                 batt_volt)
+
+        self.battery_voltage_time = time.time()
+
+        self.signal_voltage.emit(self, batt_volt)
+        self.signal_low_voltage_threshold.emit(self, lb_level)
+        self.signal_heartbeat_interval.emit(self, hb_interval)
+
+        on_done(True, "Operation complete", msg.data[5])
+
+    #-----------------------------------------------------------------------
+    def handle_ext_cmd(self, msg, on_done):
+        """Handle replies to the set_flags command.
+        Nothing to do, any NAK of failure is caught by the message handler
+        """
+        on_done(True, "Operation complete", None)
+
+    #-----------------------------------------------------------------------
+    def handle_low_batery_voltage(self, msg, on_done, voltage):
+        """Callback for handling set_low_battery_voltage() responses.
+
+        This is called when we get a response to the
+        set_low_battery_voltage() command. Update stored low bat voltage in
+        device DB and call the on_done callback with the status.
+
+        Args:
+          msg (InpStandard): The response message from the command.
+          on_done: Finished callback.  This is called when the command has
+                   completed.  Signature is: on_done(success, msg, data)
+        """
+        self.db.set_meta('low_battery_voltage', voltage)
```

Comment:
This metadata value doesn't appear to be used otherwise, though, there is an HDS:battery_low_voltage in the metadata.

Reply:
Yeah - reviewing this more closely I think I understand this better now.  I've removed this.